modules/KalturaSupport/resources/mw.kAnalony.js

```diff
@@ -314,10 +314,6 @@
 			});
 		},
 		resetPlayerflags:function(){
-			this._p25Once = false;
```

Comment:
@shalommeoded now you never reset these flags, so in change media you won't report them again

Reply:
@OrenMe While inspecting the fix on Playlist and it changed media, and the quarterly flags were showing as expected.

It seems that the session is restarting when the 'onChangeMeida' is called:
https://github.com/shalommeoded/mwEmbed/blob/bf98f379e4902ae3919d547ff9323781980b1c26/modules/KalturaSupport/resources/mw.kAnalony.js#L116

Please correct me if I have misunderstood your comment.

Reply:
reset session doesn't clear the percentiles flags. Moreover you just delted the only code that resets these flags so they won't be reset at all.

Reply:
@OrenMe So instead of deleting the flags, I have moved them into the onChangeMedia function, which will reset the flags to flase again with the onChangeMedia is called.